cmd/fleet/serve.go

```diff
@@ -645,6 +645,14 @@ the way that the Fleet server works.
 				}
 			}
 
+			if license.IsPremium() && config.Activity.EnableAuditLog {
+				if err := cronSchedules.StartCronSchedule(func() (fleet.CronSchedule, error) {
```

Comment:
I hadn't considered the use of the scheduler for an async cron, but I suppose it works perfectly when the "events" received come from the database, it already handles all the waits and so on.

Reply:
+1, plus we make sure only one fleet instance does the streaming.